mono/metadata/icall.c

```diff
@@ -9205,6 +9205,44 @@ add_internal_call_with_flags (const char *name, gconstpointer method, guint32 fl
 	}
 }
 
+/**
+* mono_dangerous_add_internal_call_coop:
+* \param name method specification to surface to the managed world
+* \param method pointer to a C method to invoke when the method is called
+*
+* Similar to \c mono_dangerous_add_raw_internal_call.
+*
+*/
+void
+mono_dangerous_add_internal_call_coop (const char *name, gconstpointer method)
+{
+	mono_add_internal_call_with_flags (name, method, TRUE);
```

Comment:
```suggestion
	add_internal_call_with_flags (name, method, MONO_ICALL_FLAGS_COOPERATIVE);
```

I think it would be better in the long term to get rid of `mono_add_internal_call_with_flags` (it seems to only be used in the pre-netcore mono native lib)

Reply:
this was probably a copy/past from mono_dangerous_add_raw_internal_call/, maybe we should fix that function as well, to use , add_internal_call_with_flags.